autopush/tests/test_endpoint.py

```diff
@@ -649,7 +620,8 @@ def test_delete_uaid(self):
     @inlineCallbacks
     def test_delete_bad_uaid(self):
         resp = yield self.client.delete(
-            self.url(router_type="test", router_token="test", uaid="invalid"),
+            self.url(router_type="test", router_token="test",
+                     uaid=uuid.uuid4().hex),
```

Comment:
tempted to suggest using a static UUID like `"0"*32`, or a comment that the generated, valid UUID is bad because there's no associated record. Might save some future confusion about why this returns a 401.